yaptide/examples/call_api_example.py

```diff
@@ -182,20 +183,26 @@ def run_simulation_with_files(session: requests.Session, example_dir, json_to_se
                 print(e)
 
 
-def run_simulation_with_rimrock(port: int = 5000):
-    """Example function running simulation on rimrock"""
-    example_dir = os.path.dirname(os.path.realpath(__file__))
-    grid_proxy_path = Path(example_dir, 'grid_proxy')
+def get_grid_proxy_file(dir_path: str) -> str:
```

Comment:
```suggestion
def get_grid_proxy(dir_path: str) -> str:
```
or
```suggestion
def read_grid_proxy_file(dir_path: str) -> str:
```
seems like a more accurate name